docs/shocks.rst

```diff
@@ -1,140 +1,134 @@
 
-2. Particle paths
------------------
+.. code-block:: bash
 
-This script will generate a file which shows the 10 most energetic particles' position and their Lorentz factor against time.
+    python3 script.py --conf file.ini
 
-To run, use the command:
+Except in case of ``plot_win_2d_shock.py``:
 
 .. code-block:: bash
 
-   python3 prtcl_path.py --conf shock_mini.ini
-  
-3. Plot shock
--------------
+    python3 plot_win_2d_shock.py --conf file.ini --lap [lap number]
 
-This script generates a four-part plot showing:
+The number after ``--lap`` specifies the simulation lap you want to view. If you want to plot *all* of the laps of a complete run, you can run
+ 
+.. code-block:: bash
 
-- A plasma density map
-- An out-of-the-plane magnetic field map (Z-direction)
-- An out-of-the-plane current density map (Z-direction)
-- Plot of 1D density and magnetic energy density
+    ./scripts.sh
 
-This produces graphs for all laps generated, unless a simulation lap is specified. Additionally, the paths of the 10 most energetic particles are shown and, if all laps are generated, the 1D density data is output for the shock velocity script.
+in the *pic-shocks* folder.
 
-To run, use the command:
 
-.. code-block:: bash
+**Example:** The output of ``plot_win_2d_shock`` should look something like this:
 
-   python3 plot_shock.py --conf shock_mini.ini [--lap lap_no]
+.. image:: https://raw.githubusercontent.com/natj/pb-utilities/60bb931396941562251ab2329fd4c07f8890e906/imgs/pic-shock1d.png 
+   :width: 800px 
 
-4. Shock Rankine-Hugoniot jump conditions
------------------------------------------
 
-This script will load in the 1D density data, and use it to find the compression ratio of the shock and subsequently the shock velocity based on the midpoint of the shock.
-A plot of position against time will be shown, and the coordinate-velocity value determined in the frame of the **downstream plasma**.
+.. note::
+   
+   The above plot is from a short 1D shock simulation (with `\gamma = 10` and `\sigma = 3`) using 2 cores. Depending on the dimensions and other parameters of your simulation the output might look slightly different.
+   
+The script plots the values in panels as functions of skin depth, `\frac{c}{\omega_p}` (scale at the bottom of the figure). Point `x = 0`, which follows the shock front, divides the plasma into *downstream* (negative `x`) and *upstream* (positive `x`) sections.
 
-To run, use the command:
+The top two panels show plasma density. If a shock has succesfully formed, you should be able to see a jump in the downstream to upstream density affected by the shock: `\frac{n_d}{n_u} \approx 1 \rightarrow 2` 
+
+Other panels include (top to bottom) `x`, `y` and `z` components of the electric field, magnetic field and electric currents. Panel just beneath `J_z` marks the MPI rank division.  Bottom three panels plot the velocity components of the particles in `x`, `y` and `z` directions accordingly.
```

Comment:
Last sentence:

The bottom four panels visualize the total velocity ``\log_10(\gamma)`` and the individual four-velocity components  ``beta_x \gamma``, ``\beta_y \gamma``, and ``\beta_z \gamma``.